src/main/java/io/vertx/core/net/KeyStoreOptionsBase.java

```diff
@@ -154,6 +156,23 @@ public KeyStoreOptionsBase setAlias(String alias) {
     return this;
   }
 
+  /**
+   * @return the alias password for a server certificate when the keystore has more than one, or {@code null}
```

Comment:
more than one password ? it reads ambiguous (could be alias)

Reply:
```suggestion
   * @return the password for the server certificate designated by {@link #getAlias()}, or {@code null}
```